recommonmark/parser.py

```diff
@@ -227,16 +228,30 @@ def inline_entity(inline):
     return entity_node
 
 
+def make_refnode(label, target, has_explicit_title):
```

Comment:
Is this logic copied from Sphinx somewhere? Should have a comment specifying where it came from.